master

ilyasshss opened the pull request.

evanilson-santana committed:
LPS-194538 Change relationship from objects for some tests that need to see the filter section in the aggregation field

Valmir-unicap committed:
LPS-197489 Transform class DomainValidatorFactory for class Util, and apply to usages

Valmir-unicap committed:
LPS-197489 Move classe FieldPredicateProviderTracker to class DefaultFilterFactoryImpl, and apply to usages

Valmir-unicap committed:
LPS-197617 Create property and prepare for next

Valmir-unicap committed:
LPS-197617 Remove method getClassName from class LayoutStructureItemMapper, and apply to usages

Commit:
LPS-198079 Merge the component DDMPermissionSupportRegistry in DDMPermissionSupportImpl

evanilson-santana committed:
LPS-194538 Remove the IsEqualToIsAvailableForLongIntegerFields since the case is being covered by CanCreateEntryWithAggregationField

evanilson-santana committed:
LPS-194538 Improvement in the test CanCreateEntryWithAggregationField

evanilson-santana committed:
LPS-194538 Remove the IsEqualToIsAvailableForIntegerFields since the case is being covered by CanCreateEntryWithAggregationField

evanilson-santana committed:
LPS-194538 Add task to add entries to ensure the aggregation field is working

evanilson-santana committed:
LPS-194538 Improvement of the test RangeIsAvailableForDateFields to add entries and ensure the aggregation is working

petershin committed:
LPS-200065 Set default deploy directory to osgi/portal

petershin committed:
LPS-200065 Readme

vicnate5 committed:
LPS-200065 Fix for LPKG builds

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

JanisZhang committed:
LPS-201042 Remove Bean RawMetadataProcessorMessageListener and corresponding Bean DestinationConfiguration from messaging-misc-spring.xml

evanilson-santana committed:
LPS-194538 Add variables for selectTitleField macro

Commit:
LPS-200453 Semantic versioning

alessiorendina committed:
COMMERCE-12756 Added new feature flag

alessiorendina committed:
COMMERCE-12756 Added new factory configuration

alessiorendina committed:
COMMERCE-12756 Added commerce payment entry refund type and registry

alessiorendina committed:
COMMERCE-12756 Added commerce payment entry refund type implementation

alessiorendina committed:
COMMERCE-12756 Added commerce payment entry refund type registry implementation

alessiorendina committed:
COMMERCE-12756 Added new comparator

alessiorendina committed:
COMMERCE-12756 Added new localized keys

alessiorendina committed:
COMMERCE-12756 Semantic versioning (gw baseline)

Mikellorza committed:
LPS-201547 Refactor code

jeffwu0724 committed:
LPS-201428 upgrade mysql-connector-j to 8.0.33 in lib

jeffwu0724 committed:
LPS-201428 upgrade mysql-connector-j to 8.0.33 in build-test.xml

jeffwu0724 committed:
LPS-201428 upgrade mysql-connector-j to 8.0.33 in modules

jeffwu0724 committed:
LPS-201428 upgrade setup.database.jar.url and setup.database.jar.sha1 in portal.properties

Mikellorza committed:
LPS-201547 Modify count assets tag usages with case-sensitive

Commit:
LRQA-83496 Add some asserts to CanCP20BeCreated Test

Commit:
LRQA-83496 Delete AccountUsersAreCreatedForCP20 Test

Commit:
LRQA-83496 Delete ObjectsAreCreatedForCP20 Test

Commit:
LRQA-83496 Delete RemoteAppsAreCreatedForCP20 Test

Commit:
LRQA-83496 Delete TestsAccountsAreCreatedForCP20 Test

Mikellorza committed:
LPS-201547 Improve code

Mikellorza committed:
LPS-201547 Move method from jsp to display context

Commit:
LPS-199195 portal-search-solr8-impl: Bump apache zookeeper

https://liferay.atlassian.net/browse/LPS-199195

jeffwu0724 committed:
LPS-201428 auto generate, by running "ant build-lib-versions"

Commit:
LPS-200552 Fix UpgradeSCSSNodeSassPatternsCheck class to attend other interpolation

Commit:
LPS-200552 Include more cases in test files

Commit:
LRQA-83496 SF

brookedalton committed:
LPS-201498 Check for other published workflow model deletions

kenjiheigel committed:
LRCI-3881 Init module as gradle project

kenjiheigel committed:
LRCI-3881 Generate playwright files

kenjiheigel committed:
LRCI-3881 Add sample test

pei-jung committed:
LPS-202050 Use browser default time input

Mikellorza committed:
LPS-201547 Improve code

Mikellorza committed:
LPS-201547 Delete redundant method

# breaking_change_report

## What portal-impl/src/com/liferay/portlet/asset/service/impl/AssetTagServiceImpl.java -->  getVisibleAssetsTagsCount(long groupId, String name)

Replace by method getVisibleAssetsTagsCount(long groupId, long classNameId, String name) passing classNameId as 0

## Why

Delete redundant method

Mikellorza committed:
LPS-201547 Delete redundant method

# breaking_change_report

## What portal-impl/src/com/liferay/portlet/asset/service/impl/AssetTagLocalServiceImpl.java --> getTagsSize(long groupId, String name)

Replace by method getTagsSize(long groupId, long classNameId, String name) passing classNameId as 0

## Why

Delete redundant method

Mikellorza committed:
LPS-201547 Delete redundant method

# breaking_change_report

## What portal-impl/src/com/liferay/portlet/asset/service/persistence/impl/AssetTagFinderImpl.java --> countByG_N(long groupId, String name)

Replace by method countByG_C_N(long groupId, long classNameId, String name) passing classNameId as 0

## Why

Delete redundant method

Mikellorza committed:
LPS-201547 buildService

Mikellorza committed:
LPS-201547 Adapt usages of deleted methods

Mikellorza committed:
LPS-201547 baseline

kenjiheigel committed:
LRCI-3881 Add playwright docker macrodefs

kenjiheigel committed:
LRCI-3881 Add playwright batch

kenjiheigel committed:
LRCI-3881 Add playwright test suite

susanchen3 committed:
LPS-201852 Temp ignore until LPS-195673 is fixed

DiegoHu97 committed:
LPS-201916 Check if the component is mounted before setting state

brookedalton committed:
LPS-200484 If publication has been published user should not see conflict

smottal committed:
COMMERCE-12431 Feature flag description

smottal committed:
COMMERCE-12431 Autogenerated Code (buildLang)

izaera committed:
LPS-201890 Don't route CX resources through combo servlet

MumenTayyem committed:
LPS-197318 - Adding fragments

DiegoHu97 committed:
LPS-202014 Add title to back button when importing display page templates

Commit:
LPS-196765 To use PG access

DiegoHu97 committed:
LPS-202023 Replace AUI() usages in push-notifications-web

Commit:
LPS-196765 Fix typo

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 To use PG access

Commit:
LPS-196765 SF

davisaints committed:
LPS-201316 Adjust incorrect syntax in getObjectFieldERC macro

Commit:
LPS-196765 Fix mistake

DiegoHu97 committed:
LPS-202132 Replace AUI() when scheduling a publication

DiegoHu97 committed:
LPS-202132 Replace AUI() in a publication's comment

kenjiheigel committed:
LRCI-3881 Format & update playwright config

Commit:
LPS-196765 Revert

This reverts commit b45fd4c2a3803bff1965f0a7e5958c6a665c0f4a.

michaelhashimoto committed:
LRCI-3890 Shift build parameters into the build object definition

michaelhashimoto committed:
LRCI-3890 Add parameters to the build information page

michaelhashimoto committed:
LRCI-3890 Remove all references to the build entity parameters

pei-jung committed:
LPS-201498 Use ctDisplayRendererRegistry to get the correct ctCollectionId

CAustin582 committed:
LPS-202078 Remove unused service

kenjiheigel committed:
LRCI-3881 Rename package

kenjiheigel committed:
LRCI-3881 Auto SF

michaelhashimoto committed:
LRCI-3890 Fixes after deployment to LXC

michaelhashimoto committed:
LRCI-3890 Move the build parameters into its own panel

JanisZhang committed:
LPS-202036 Fix test and remove redundant test logic

marianoalvarosaiz committed:
LPS-202081 Use the already calculated default language for language selector selected language.

shuyangzhou committed:
LPS-202211 Switch to use SimpleURLPatternMapper. DynamicSizeTrieURLPatternMapper/StaticSizeTrieURLPatternMapper are way over designed. Out of box we only have 483 URL patterns, they wasted about 70MB heap for empty long arrays.

haydenzheng314 committed:
LPS-202093 Replace non-blocking ServiceProxyFactory usages with Snapshot in class AggregateFilter

JanisZhang committed:
LPS-202096 Replace blocking ServiceProxyFactory with Snapshot in class AutoBatchPreparedStatementUtil

alessiorendina committed:
COMMERCE-12756 Export package

alessiorendina committed:
COMMERCE-12756 BuildLang

MumenTayyem committed:
LPS-197318 - adding fragments

Commit:
LPS-201439 Modify cases to make them stable

Commit:
LPS-196765 Revert since it's a private host

Commit:
LPS-196765 Revert to use PG access

This reverts commit 437dfe90c61ea0f5fde5a7c3fa381b86147b7620.

Commit:
LPS-196765 Revert to use PG access

This reverts commit a22ed154

lfbesada committed:
LPS-202222 Disallows to convert to page template for no-content type layouts

Commit:
ISSD-1743 Add offering types for DXP Apps

p2kmgcl committed:
LPS-200687 Fetch should return a promise

p2kmgcl committed:
LPS-200687 Extract useSafeReducer

p2kmgcl committed:
LPS-200687 Types

p2kmgcl committed:
LPS-200687 Wait until component has loaded

marcogalluzzi committed:
LPS-202131 No need to have this method as public

p2kmgcl committed:
LPS-200687 Add button label

marcogalluzzi committed:
LPS-202131 Fix: updated visibility of asset entry

ealonso committed:
LPS-201623 Revert "LPS-200805 Removes LPS-180328 Feature Flag" since we want to keep it at least one more version

Luizcarlosqueiroz committed:
LPS-201555 Fix notifications type path

Luizcarlosqueiroz committed:
LPS-201555 Make notification checkbox path more resilient

DiegoHu97 committed:
LPS-202132 Replace AUI() when navigate

saramart81 committed:
COMMERCE-12756 Add functional tests

Commit:
LPS-196765 Revert

Commit:
LPS-196765 Revert To use PG access

This reverts commit 70bea3db64c05131d4a832ca6d1c9cd6bd4d8c21.

pyoo47 committed:
LRCI-3888 Reorganize ParallelExecutor to allow users to process in parallel, multiple lists of callables that are run sequentially.

Commit:
ISSD-1743 Create Enums for App Build Page

BarbaraCabrera committed:
LPS-202150 Check if the ddmStructureId is less or equal to zero

kenjiheigel committed:
LRCI-3881 Stabilize connection by using Jenkins node hostname with the playwright container

DiegoHu97 committed:
LPS-202132 Replace AUI() when searching a Publication

ealonso committed:
LPS-194261 Removes LPS-192053 Feature Flag

kenjiheigel committed:
LRCI-3881 Skip playwright dir for ModulesStructureTest

gaborlovasdotcom committed:
LRQA-84469 Update macro to support multiple tomcat instances

pyoo47 committed:
LRCI-3888 Fix broken ParallelExecutor usages

Commit:
ISSD-1743 Improve enums and change for global file

Commit:
ISSD-1743 Add i18n for translations

pyoo47 committed:
LRCI-3888 Set http connection and read timeouts to 5 minutes

DiegoHu97 committed:
LPS-201916 Remove required prop since the variable is not used

DiegoHu97 committed:
LPS-201916 Change with proper attribute

pyoo47 committed:
LRCI-3888 The first step of run<Status>() should not be to set the status to a status that it already has.

pyoo47 committed:
LRCI-3888 Remove debug code

pyoo47 committed:
LRCI-3888 Don't create a parallel task for a build that is complete

Commit:
ISSD-1743 Add disabled for fully-managed dxp type

Commit:
ISSD-1743 Add translation for offering types titles and descriptions

Commit:
ISSD-1743 Fix title and tooltip space

pyoo47 committed:
LRCI-3888 Don't call getBuildMessage() unless the status has changed

davisaints committed:
LPS-201316 Use getPortalURL macro instead of getDefaulPortalURL

shuyangzhou committed:
LPS-202211 Fix unit test

Commit:
LPS-202198 Type 12 hour UTC time

Commit:
LPS-202177 Remove duplicates macro

Commit:
LPS-202177 Remove property that skip testcase on virtual instance

Commit:
LPS-202177 Add saves to avoid errors on CI

gaborlovasdotcom committed:
LRQA-84469 Update test case file

interaminense committed:
LRAC-15075 Follow up unit tests

brianchandotcom committed:
Revert "LPS-200065 Readme"

This reverts commit 24166856bd639321b3804cfb7846b03cf38b384a.

brianchandotcom committed:
Revert "LPS-200065 Set default deploy directory to osgi/portal"

This reverts commit 3808e5d5dba5a7554bb1ef14a608c2c3d4149f43.

brianchandotcom committed:
LPS-200065 prep next

brianchandotcom committed:
LPS-201547 Rename

interaminense committed:
LRAC-15057 Create unit tests for Page Path card

interaminense committed:
LRAC-15057 Create unit tests for Filter By Segment component

interaminense committed:
LRAC-15057 Remove debug

pyoo47 committed:
LRCI-3888 Don't look for the build in the queue immediately

fmarcio committed:
LRAC-15075 update snapshots

michaelhashimoto committed:
LRCI-3890 SF

pyoo47 committed:
LRCI-3888 Check running task count + remaining task count

michaelhashimoto committed:
LRCI-3902 Create a Jethr0 Message Listener that will send updates to Jethr0BuildUpdater

pyoo47 committed:
LRCI-3888 Increase timeout to 15 minutes

pyoo47 committed:
LRCI-3888 Count actual running callables

brianchandotcom committed:
LRCI-3888 Wordsmith

brianchandotcom committed:
LRCI-3888 prep next

Commit:
LPS-202209 Remove tear down block

Commit:
LPS-202209 Modify case to make it stable

Commit:
LPS-202210 Remove tear down block

Commit:
LPS-202210 Modify case to make it stable

Commit:
LPS-200453 Make PortletToolbar not a spring bean and provide the instance through filed INSTANCE.

JanisZhang committed:
LPS-201042 Move RawMetadataProcessorMessageListener from portal-impl to module document-library-service

JanisZhang committed:
LPS-201042 Make RawMetadataProcessorMessageListener as a component and register Destination in activate method.

Songyuewen committed:
LPS-202107 Update message

JamesAGarcia committed:
COMMERCE-12431 wordsmiths feature flag description

Commit:
LPS-201911 Update the step to review changes

Commit:
LPS-202221 update maven-war-plugin version

MumenTayyem committed:
LPS-197318 - Fragments SF

Commit:
LPS-199849 Add case to filter attachment from computer by file type

MumenTayyem committed:
LPS-197318 - Fragments SF

JamesAGarcia committed:
LPS-191150 wordsmiths feature flag description

lfbesada committed:
LPS-201821 Sets in the service context the user who is going to be the creator of the new data.

We are using the target layout creator at first instance to maintain the current behavior but adding as fallbacks the source layout user and the current user to ensure that is always an existing user

lfbesada committed:
LPS-201821 Uses the user as creator of all data

Songyuewen committed:
LPS-201357 Add a step wait for published

Commit:
LPS-199849 Add case to filter attachment from documents and media by file type

Commit:
LPS-199849 Add case to assert cannot drag and drop invalid file type to documents and media modal

ealonso committed:
LPS-202237 Removes LPS-196766 Feature Flag

lfbesada committed:
LPS-201821 Adds an integration tests

brianchandotcom committed:
LPS-202096 as used

liferay-translation committed:
LPS-77699 Update Translations

Commit:
LPS-202221 update css builder for module

Commit:
LPS-202221 restore css builder for theme template

Commit:
LPS-202221 update service builder to 1.0.469 for maven

Songyuewen committed:
LPS-202008 Update page name

bryceosterhaus committed:
LPS-201375 - Explicitly check 'true' from the cookie consent type

marcogalluzzi committed:
LPS-202242 Make trashHelper available to all portlets

Songyuewen committed:
LPS-202006 Add site before create publications

Commit:
ISSD-1743 SF

gaborlovasdotcom committed:
LRQA-84469 Relocate file as it's referenced in multiple test case files

gaborlovasdotcom committed:
LRQA-84469 Update file location in build-test-oauth2.xml

brianchandotcom committed:
LPS-202237 SF

DiegoHu97 committed:
LPS-202085 Convert Bytes into MegaBytes

ealonso committed:
LPS-202249 Discard code under LPS-149256

brianchandotcom committed:
LPS-200065 prep next

Songyuewen committed:
LPS-202024 Wait for page load

Songyuewen committed:
LPS-200456 Assert success message in macro

Mikellorza committed:
LPS-202248 Delete unused finder

MumenTayyem committed:
LPS-197318 - Fragments SF

Commit:
ISSD-1746 The Create License Key button does not work

Mikellorza committed:
LPS-202248 buildService. Delete unused finder.

# breaking_change_report

## What portal-kernel/src/com/liferay/asset/kernel/service/persistence/AssetTagUtil and portal-kernel/src/com/liferay/asset/kernel/service/persistence/AssetTagPersistence

portal-kernel/src/com/liferay/asset/kernel/service/persistence/AssetTagUtil.java and portal-kernel/src/com/liferay/asset/kernel/service/persistence/AssetTagPersistenc.java. Now there are not include the methods findByG_LikeN,findByG_N_First,findByG_N_Last,findByG_N_PrevAndNext,removeByG_N and countByG_N

## Why

These changes are because of there are not use in the portal.

----

victorg1991 committed:
LPS-197787 Check if the component is mounted

Commit:
ISSD-1746 Use DXP order type instead of Cloud

victorg1991 committed:
LPS-197787 Add tests for RulesModal

Commit:
ISSD-1746 SF

brianchandotcom committed:
LPS-197318 Sublime sort

brianchandotcom committed:
LPS-197318 Can you rename this file

brianchandotcom committed:
LPS-197318 I went to portal-impl and ran 'ant format-source-current-branch'

Mikellorza committed:
LPS-202248 Restore case-sensitive=false in order to provide the same behavior for all databases. Then in the AssetTagLocalServiceImpl and AssetTagServiceImpl should be taken into account the case-sensitive where we have to take account case-sensitive

brianchandotcom committed:
LPS-197318 Manual SF

brianchandotcom committed:
LPS-197318 I like {

Mikellorza committed:
LPS-202248 buildService

brianchandotcom committed:
LPS-197318 Sort methods. Please sort vars if possible for me in a follow up PR.

ealonso committed:
LPS-202257 Sets default value for _fragmentElementId to avoid NPE, like we do in DefaultFragmentRendererContext

ealonso committed:
LPS-202257 No needed since we already have a default value

MumenTayyem committed:
LPS-197318 - Fragments Follow Up

brianchandotcom committed:
LPS-197318 ran SF

brianchandotcom committed:
LPS-197318 manual SF

DiegoHu97 committed:
LPS-202264 Remove not used file

Commit:
LPS-198962 Decrease validation delay

brianchandotcom committed:
LPS-201821 SF

Commit:
LPS-196765 Revert

Mikellorza committed:
LPS-202248 Make finder case-insensitive

pyoo47 committed:
LRCI-3888 Increase number of threads

brianchandotcom committed:
LRCI-3902 prep next

brianchandotcom committed:
LPS-200552 prep next

brianchandotcom committed:
LPS-200552 prep next

Commit:
ISSD-1746 Use Loading, SWR, Simplify

brianchandotcom committed:
LRCI-3888 prep next

petershin committed:
Revert "LPS-87890 drop hours, minutes, and seconds"

This reverts commit d089b938a9abc63c6250ee533b94b3e540e300b5.

petershin committed:
LPS-87890 Human readability

Commit:
LPS-202184 Add property to skip virtual instance and organize the testcase

michaelhashimoto committed:
LRCI-3907 Add logic that will check for jars if no lpkg files exist

Commit:
LPS-202184 Fix title field the CanViewTypeOnFieldEntry

pyoo47 committed:
LRCI-3888 Default timeout should be in seconds

Commit:
LPS-202258 Fix testcase with wrong label

pyoo47 committed:
LRCI-3888 Add getSubmittedTaskCount()

brianchandotcom committed:
LPS-87890 prep next

drewbrokke committed:
LPS-202307 gradle-plugins-workspace: adds client-extension.schema.json

brianchandotcom committed:
LRCI-3907 prep next

brianchandotcom committed:
LPS-202307 Wordsmith

brianchandotcom committed:
LPS-202221 prep next
